public/index.html

```diff
@@ -129,12 +130,17 @@
               </div>
               <a href="index.html" aria-current="page" class="brand w-nav-brand w--current"><img src="images/Logo.svg" loading="lazy" alt=""></a>
               <nav role="navigation" class="nav-menu w-nav-menu">
-                <a href="./support.html" class="nav-link white w-nav-link" onclick='viewOnDevopsInABoxGAEvent("Nav")'>#DevOpsInABox 🎁</a>
+                <a href="./support.html" class="nav-link white w-nav-link" onclick='viewOnDevopsInABoxGAEvent("Nav")'>#DevOpsInABox</a>
                 <a href="https://docs.devtron.ai/" class="nav-link white w-nav-link" target="_blank">Docs</a>
                 <a href="team.html" aria-current="page" class="nav-link white w-nav-link">Team</a>
                 <a href="careers.html" class="nav-link white w-nav-link">Careers</a>
                 <a href="https://devtron.ai/blog/" class="nav-link white w-nav-link" target="_blank">Blog</a>
-                <a href=" https://preview.devtron.ai/dashboard/" class="demo-button tab-view" target="_blank" onclick="demoHeader('homepage')">Demo <img src="images/arrow-right.svg"></a>
```

Comment:
is removal of onclick="demoHeader('homepage')" is expected?

Reply:
Replaced with Try Devtron

Reply:
Added